### PR TITLE
DXKulture: Change domain

### PR DESCRIPTION
**Pull request by kmdevops**
Changes:
1. Change domain from `kulture.media` to `dxkulture.com`
2. Change contact email address from `devops@kulture.media` to `devops@dxkulture.com`

### CODE REVIEW

File: static/bidder-info/dxkulture.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
-endpoint: "https://ads.kulture.media/pbs"
+endpoint: "https://ads.dxkulture.com/pbs"
```
Comment (onkarvhanumante) [inline static/bidder-info/dxkulture.yaml:1]: endpoint is reachable

```
% curl -i https://ads.kulture.media/pbs
HTTP/1.1 405 Method Not Allowed
Date: Mon, 18 Dec 2023 05:50:39 GMT
Content-Length: 0
Connection: keep-alive
Vary: Origin
Strict-Transport-Security: max-age=15724800; includeSubDomains
```

File: static/bidder-info/dxkulture.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 maintainer:
-  email: "devops@kulture.media"
+  email: "devops@dxkulture.com"
```
Comment (onkarvhanumante) [inline static/bidder-info/dxkulture.yaml:3]: @kmdevops  prebid team has sent email to verify above address. Requesting to respond with "received" message on email thread

File: static/bidder-info/dxkulture.yaml [hunk 3]
```diff
@@ -12,5 +12,5 @@ capabilities:
       - video
 userSync:
   redirect:
-    url: "https://ads.kulture.media/usync?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&cb={{.RedirectURL}}"
+    url: "https://ads.dxkulture.com/usync?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&cb={{.RedirectURL}}"
```
Comment (onkarvhanumante) [inline static/bidder-info/dxkulture.yaml:15]: url is redirecting

**Review by onkarvhanumante — APPROVED** [review]

**Review by gargcreation1992 — APPROVED** [review]
g2g after email verification!

**Pull request merged**